api/CSSMediaRule.json

```diff
@@ -5,42 +5,42 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSMediaRule",
         "support": {
           "chrome": {
-            "version_added": "45"
+            "version_added": "1"
           },
           "chrome_android": {
-            "version_added": "45"
+            "version_added": "1"
```

Comment:
I see the linter is saying, *“version_added: "1" is NOT a valid version number for chrome_android”*